website/docs/en/config/index.mdx

```diff
@@ -2,6 +2,10 @@
 
 Rspack provides configurations similar to webpack. This chapter will show you how to use the Rspack configuration.
 
+:::info
+`webpack-merge` also applies to Rspack.
```

Comment:
Can add add a `Merge Configurations` section and provide an example?

Like this: https://rsbuild.dev/guide/basic/configure-rsbuild#merge-configurations

Reply:
And link to the repo: https://github.com/survivejs/webpack-merge

Reply:
Fine, modified to `Merge Configurations` section.